code/game/objects/structures/crates_lockers/closets/secure/engineering.dm

```diff
@@ -56,6 +50,12 @@
 	for(var/i in 1 to 3)
 		new /obj/item/multitool(src)
 
+/obj/structure/closet/secure_closet/engineering_electrical_empty

```

Comment:
Make these subtypes of the real things, but with PopulateContents() returning nothing.

Reply:
Once I find out how to make PopulateContents() not affect the subtype I will, so this PR is going to be open for a few days.

Reply:
```
/obj/structure/closet/secure_closet/engineering_electrical/empty/PopulateContents()
	return
```

Is it too hard, you little dum dum?